src/theme/_fonts.scss

```diff
@@ -1,2 +1,2 @@
-$primaryFontFamily: 'Raleway', sans-serif;
+$primaryFontFamily: 'Salesforce Sans', sans-serif;
```

Comment:
@ankitecd 
This should be overridden on tenant config, and not on app level configuration.

Default font on other tenant is still Raleway.

Also fonts needs to cover Bold and other texts so additional variants need to be loaded as well.